src/server/controllers/match.controller.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const matchSchema = require('mongoose').model('match');
+import mailer from 'nodemailer';
+
+/*
+ * Methods about mentoring request, accept or reject including E-mail Service
+ */
+
+let matchData = new matchSchema();
+
+let sampleSuccessResult = {
+  successCode: 1,
+  matchData: matchData,
```

Comment:
to change sample result dynamically according to reqest parameter,
**function requestMentoring(..)** change the content of first **matchData**.
and **sampleSuccessResult** take that values